src/rf/apps/workers/process.py

```diff
@@ -71,8 +77,13 @@ def start_polling(self):
     def handle_message(self, record):
         # Parse record fields.
         try:
-            job_type = record['eventName']
-            event_source = record['eventSource']
+            if 'eventName' in record:
+                job_type = record['eventName']
+                event_source = record['eventSource']
+            elif 'stage' in record:
+                job_type = record['stage']
+            else:
```

Comment:
In future implementations, each job should be responsible for parsing message content, so we don't have to hard-code particular rules about message format in the queue processor. Without prior knowledge, it's unclear why `job_type` could be set from either `record['eventName']` or `record['stage']` and which jobs expect which format?